script.js

```diff
@@ -1,12 +1,116 @@
-//You can edit ALL of the code here
+//TV Show Project
+
+const allEpisodes = getAllEpisodes();
+const episodeContainer = document.getElementById("cardContainer");
+const rootElem = document.getElementById("root");
+const searchInput = document.getElementById("searchEpisode");
+const episodesCount = document.getElementById("searchResult");
+const viewAllEpisodesButton = document.querySelector("#viewAllEpisodes");
+
 function setup() {
-  const allEpisodes = getAllEpisodes();
   makePageForEpisodes(allEpisodes);
 }
 
+// display all episodes //
+
 function makePageForEpisodes(episodeList) {
-  const rootElem = document.getElementById("root");
-  rootElem.textContent = `Got ${episodeList.length} episode(s)`;
+  rootElem.innerHTML = "";
+  episodeList.forEach((episode) => createCard(episode));
+  episodesCount.innerText = `Displaying ${episodeList.length}/${allEpisodes.length}`;
+}
+
+// create card //
+
+function createCard(episode) {
```

Comment:
Smart idea moving this process into another function. It can make your code a little easier to read and easier to handle :) good job!